draft-toorop-dnsop-dns-catalog-zones.md

```diff
@@ -194,9 +194,11 @@ For example, if a catalog zone lists three zones "example.com.",
 
 where `<m-unique-N>` is a label that uniquely tags each record in the collection.
 
-The reason for the `<m-unique-N>` label is that catalog zones could contain a larger number of zones.
-The number of zones might be larger than that what can be conveyed in a single "zones" RRset in a zone transfer.
-Uniquely tagging each zone makes sure the list can be split over multiple transfer messages.
+Having a large number of member zones in a single RRset may cause the RRset to be too large to be conveyed via DNS messages which make up a zone transfer.
+Having the zones uniquely tagged with the `<m-unique-N>` label ensures the list of member zones can be split over multiple DNS messages in a zone transfer.
+
+The `<m-unique-N>` label also enables the state for a zone to be reset. (see #zonereset)
+As long as no zone state needs to be reset at the authoritative nameservers, the unique label associated with a zone MUST remain the same.
```

Comment:
I wonder if 'SHOULD' is good enough here, and I wonder if MUST would be an annoying constraint for some setups?